opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/TestSpringWebTracing.java

```diff
@@ -63,15 +77,23 @@ public void testControllerTracing() {
 
   @Test
   public void testRestTemplateTracing() {
-    restTemplate.getForEntity("http://www.example.com", String.class);
+    restTemplate.getForEntity(getUrl("/notTraced"), String.class);
     await().until(() -> mockTracer.finishedSpans().size() == 1);
-    assertEquals(1, mockTracer.finishedSpans().size());
+    List<MockSpan> mockSpans = mockTracer.finishedSpans();
+    assertEquals(1, mockSpans.size());
+    assertEquals(Tags.SPAN_KIND_CLIENT, mockSpans.get(0).tags().get(Tags.SPAN_KIND.getKey()));
```

Comment:
I might still be missing a coffee or two, but if the endpoint is *not* to be traced, why do you expect a span to exist?